tests/app/celery/test_lookup_va_profile_id_task.py

```diff
@@ -297,3 +304,38 @@ def test_should_not_call_callback_on_retryable_exception(client, mocker, notific
 
     mocked_mpi_client.get_va_profile_id.assert_called_with(notification)
     mocked_check_and_queue_callback_task.assert_not_called()
+
+
+def test_should_permanently_fail_and_clear_chain_when_technical_failure_exception(client, mocker, sample_notification):
```

Comment:
I didn't mark all the tests in this review, but all tests that were modified should have updated names to match our naming convention.